ushahidi/platform-client-mzima
# [Outreachy Task Submission] Add Keyboard Support For Add Post Modal

Resolves [#4897](https://github.com/ushahidi/platform/issues/4897)
**Description**

Currently, the Add Post Dialog is not accessible using keyboard and screen readers. The following are the accessibility issues identified with this section:

1. Survey Items are not announced properly
2. Survey Items do not receive focus when tabbing
3. Survey items do not respond to click events

**How To Test**
Try to add a post using just the keyboard, you'll notice the following:

1. Survey items now receive focus
2. Survey items are announced properly
3. Survey items now respond to the enter click event

@Angamanga Please help review 🙏🏽
